test/runtests.jl

```diff
@@ -68,6 +68,173 @@ end
                    8.848886704358202, 2.8697881579099085, 11.15107375630744, 11.8392578374927])
 end
 
+@testset "rankdeficient-glm" begin
+    df = CSV.read("rankdeficient-glm.csv")
+
+    OLS_1 = fit(
+        LinearModel,
+        @formula(label_1 ~ 1 + feature_1 + feature_2 + feature_3),
+        df,
+        true)
+    Test.@test coef(OLS_1)[1] ≈ 0.4814822002921986
+    Test.@test coef(OLS_1)[2] ≈ 0.0
+    Test.@test coef(OLS_1)[3] ≈ 7.100773448428529e-6
+    Test.@test coef(OLS_1)[4] ≈ 4.306733126416892e-6
+    Test.@test stderr(OLS_1)[1] ≈ 0.013269599196573958
+    Test.@test isnan(stderr(OLS_1)[2])
+    Test.@test stderr(OLS_1)[3] ≈ 8.60301517985726e-6
+    Test.@test stderr(OLS_1)[4] ≈ 3.452441071366933e-6
+    Test.@test coeftable(OLS_1).cols[4][1].v ≈ 6.294030157059516e-271
```

Comment:
Does this really test something different from the line above? `coeftable` is just a convenience function, it shouldn't be needed to test it here. Same below.